internal/config/cache/config.go

```diff
@@ -115,11 +118,20 @@ func parseCacheDrives(drives string) ([]string, error) {
 			endpoints = append(endpoints, d)
 		}
 	}
-
+	isCICD := os.Getenv("MINIO_CI_CD") != "" || os.Getenv("CI") != ""
 	for _, d := range endpoints {
 		if !filepath.IsAbs(d) {
 			return nil, config.ErrInvalidCacheDrivesValue(nil).Msg("cache dir should be absolute path: %s", d)
 		}
+		if runtime.GOOS != "windows" && !isCICD {
```

Comment:
IsRootDisk() handles windows so its  okay to only have this check when CICD is false.